samples/kubernetes-event-sources/k8s-events-sa.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: k8s-events-rbac
+subjects:
+  - kind: ServiceAccount
+    # Reference to upper's `metadata.name`
+    name: default
+    # Reference to upper's `metadata.namespace`
+    namespace: default
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
a non cluster-admin role will be better or at least a comment for such desire

Reply:
hehehe, oh yes :) hence the WIP :) Just wanted to get this working, thanks!